dm-manager/pkg/api/dm-manager/dm-manager.proto

```diff
@@ -0,0 +1,72 @@
+// SPDX-FileCopyrightText: (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package device_management;
+
+option go_package = "dm-manager/pkg/api/dm-manager;dm_manager";
+
+// Enum for operation type
+enum OperationType {
+    ACTIVATE = 0;   
+    DEACTIVATE = 1;
+}
+
+// Enum for AMT status
+enum AMTStatus {
+    ENABLED = 0;
+    DISABLED = 1;
+}
+
+// Enum for activation result status.
+enum ActivationStatus {
+    PROVISIONED = 0;
+    FAILED = 1;
+}
+
+// Service for Device Management.
+service DeviceManagement {
+    // PMA reports AMT status to dm manager.
+    rpc ReportAMTStatus(AMTStatusRequest) returns (AMTStatusResponse);
+
+    // PMA sends activation request to dm manager.
+    rpc SendActivationRequest(ActivationRequest) returns (ActivationDetailsResponse);
```

Comment:
Based on the flow diagram in [the design proposal](https://github.com/open-edge-platform/edge-manageability-framework/blob/53ecd97959f26c266b16be81200b14c012f393f6/design-proposals/vpro-device.md#proposal), PMA will not know whether a device should be activated or deactivated until the manager informs it. In that case, this API should be a way for the PMA to detect whether it needs to activate a device or not.

Can you rename it to `RetrieveActivationRequest` to clarify that it is the PMA checking whether a device should be activated based on what the dm-manager knows please?